libs/components/src/alert/vl-alert.component.ts

```diff
@@ -1,167 +1,94 @@
-import { BaseElementOfType, webComponent } from '@domg-wc/common-utilities';
+import { html, PropertyDeclarations, TemplateResult, CSSResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { BaseLitElement } from '@domg-wc/common-utilities';
 import { alertStyle, iconStyle } from '@domg/govflanders-style/component';
-import { accessibilityStyle, resetStyle } from '@domg/govflanders-style/common';
-import { VlAlertClosedEvent } from './vl-alert-closed-event';
-
-@webComponent('vl-alert')
-export class VlAlert extends BaseElementOfType(HTMLElement) {
-    static get _observedAttributes() {
-        return ['icon', 'title', 'closable', 'type', 'size'];
-    }
-
-    constructor() {
-        super(`
-          <style>
-            ${resetStyle}
-            ${iconStyle}
-            ${alertStyle}
-            ${accessibilityStyle}
-          </style>
-          <div id="alert" class="vl-alert" role="alert">
-            <div id="content" class="vl-alert__content">
-              <p id="title" class="vl-alert__title">
-                <slot name='title'></slot>
-              </p>
-              <div id="message" class="vl-alert__message">
-                <slot id="messages-slot"></slot>
-              </div>
-              <div id="actions" class="vl-alert__actions">
-                <slot id="actions-slot" name="actions"></slot>
-              </div>
+import { accessibilityStyle, resetStyle, markStyle } from '@domg/govflanders-style/common';
```

Comment:
ik denk dat het gevaarlijk is hier al die `markStyle` te importeren vooraleer we nieuwe release doen van de `@domg/govflaners-style`-package

dit zal volgens mij de build blokkeren

Reply:
misschien dan best deze PR als draft markeren tot de nieuwe versie van @domg/govflanders-style er is - en pas nadat dat beschikbaar is die draft er af halen

Reply:
In draft gezet